app/controllers/search_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SearchController < ApplicationController
+  def index
+    @nearest_station = NrelFacade.create_fuel_station(params[:location])
+    @directions = MapquestFacade.create_directions(params[:location])
```

Comment:
Per the note left in your Mapquest Service: you could consider adding a destination param to your MaquestFacade and then call it from here like this:

`MapquestFacade.create_directions(params[:location], @nearest_station.address)`

so that you don't have to call the NREL facade in your Mapquest Service